tests/api/api/test_artifacts.py

```diff
@@ -394,3 +396,33 @@ def test_list_artifact_with_multiple_tags(db: Session, client: TestClient):
                 ignore_order=True,
             )
         ) == {}
+
+
+def test_store_artifact_calculate_size(db: Session, client: TestClient):
```

Comment:
u can take `tmp_path` fixture instead of creating temp file and deleting it at the end